addons/account/i18n/account.pot

```diff
@@ -4,10 +4,10 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Odoo Server 15.0\n"
+"Project-Id-Version: Odoo Server 15.0+e\n"
```

Comment:
could you get rid of the +e in community modules?

Reply:
Re-exported them all with only community as an addons path